packages/orderbook/src/orderbook.ts

```diff
@@ -99,6 +100,11 @@ export class Orderbook {
     );
   }
 
+  // Default order expiry to 2 years from now
+  static defaultOrderExpiry(): Date {
+    return new Date(Date.now() + 1000 * 60 * 60 * 24 * 365 * 2);
```

Comment:
Is this what we've always used?

Reply:
i see that it is

Reply:
Yep but it was hard coded in multiple places, I've consolidated it to be a static function.